src/dtemplate.d

```diff
@@ -633,9 +633,10 @@ public:
         return false; // don't scan uninstantiated templates
     }
 
-    override const(char)* kind()
+    override const(char)* kind() const
     {
-        return (onemember && onemember.isAggregateDeclaration()) ? onemember.kind() : cast(char*)"template";
+	auto onemember = cast(Dsymbol)this.onemember;	// strip const for the moment
```

Comment:
Because that would be a gigantic change if you wish to avoid more casts. Note how many of these smaller PRs eliminate casts as well as adding them.